cluster/images/etcd/migrate/rollback_v2.go

```diff
@@ -185,9 +185,7 @@ func traverseMetadata(head *store.NodeExtern, handleFunc func(*store.NodeExtern)
 
 		handleFunc(n)
 
-		for _, next := range n.Nodes {
```

Comment:
not only styling but it's also improving performance codewise (well, a bit)

/lgtm